e2e/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "e2e",
+  "version": "1.0.0",
```

Comment:
Doesn't really matter, but the other packages are at 0.1.0.  Not sure if there's any versioning strategy needed.

Reply:
I'll make it match the others... we should think about versioning...